plugins/module_utils/foreman_helper.py

```diff
@@ -525,7 +525,11 @@ def fail_from_exception(self, exc, msg):
         fail = {'msg': msg}
         if isinstance(exc, requests.exceptions.HTTPError):
             try:
-                fail['error'] = exc.response.json()['error']
+                response = exc.response.json()
```

Comment:
this still can raise an exception and thus fuck up the traceback, improvements welcome

Reply:
You mean when the response is not json deserializable?

Reply:
Yes, response.json will throw a valueerror

Reply:
For the occasional 503?
But there is a `try` to catch that. I think we are fine.

Reply:
Oh, i wasn't verbose enough. Sorry.

Fail json will output a traceback if there is one (without us doing anything). Now if a ValueError happens here, the traceback will be for that and not from the exception we're handling.